File: Capitulo_3/JavaScript/C_Exercicios_Propostos_pag_79_ate_82/exercicio-proposto-4.js
const prompt = require('prompt-sync')();

const usuario = prompt('Digite seu usuario:')
const senha = prompt('Digite sua senha:')

if (usuario == 'procopio' && senha == '12345'){
    console.log('Seja bem vindo!')
}else if(usuario == 'paiva' && senha == '54321'){
    console.log('Seja bem vindo!')
}else{
    console.log('Usuário e senha não conferem')
}

File: Capitulo_3/Python/C_Exercicios_Propostos_pag_79_ate_82/exercicio_proposto_4.py
print('Bem vindo ao login!')
usuario = input('Digite seu usuário:')
senha = input('Digite sua senha:')

if usuario == 'procopio' and senha == '12345':
    print('Seja bem vindo!')
elif usuario == 'paiva' and senha == '54321':
    print('Seja bem vindo!')
else:
    print('Usuário e senha não conferem')